src/commands/registries/deleteRemoteImage.ts

```diff
@@ -4,16 +4,21 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { DialogResponses, IActionContext, UserCancelledError, parseError } from '@microsoft/vscode-azext-utils';
-import { RegistryV2DataProvider, V2Tag } from '@microsoft/vscode-docker-registries';
+import { CommonTag } from '@microsoft/vscode-docker-registries';
 import { ProgressLocation, l10n, window } from 'vscode';
 import { ext } from '../../extensionVariables';
 import { UnifiedRegistryItem } from '../../tree/registries/UnifiedRegistryTreeDataProvider';
 import { getImageNameFromRegistryTagItem } from '../../tree/registries/registryTreeUtils';
 import { registryExperience } from '../../utils/registryExperience';
 
-export async function deleteRemoteImage(context: IActionContext, node?: UnifiedRegistryItem<V2Tag>): Promise<void> {
+export async function deleteRemoteImage(context: IActionContext, node?: UnifiedRegistryItem<CommonTag>): Promise<void> {
     if (!node) {
-        node = await registryExperience(context, ext.registriesTree, { include: ['genericRegistryV2Tag', 'azureContainerTag'] }, false);
+        node = await registryExperience(context, ext.registriesTree, { include: 'commontag', exclude: ['githubRegistryTag', 'dockerHubTag'] }, false);
+    }
+
+    const provider = node.provider as unknown as CommonTag;
```

Comment:
I assume it's a mistake, casting the node _provider_ to `CommonTag`? As opposed to `CommonRegistryDataProvider` or similar?

Reply:
guess I should've taken your advice and go rest haha